Source/DafnyPipeline.Test/Issue1355.cs

```diff
@@ -16,16 +18,11 @@ public class Issue1355 {
     [Fact]
     public void Test() {
       var options = DafnyOptions.Create();
-      ErrorReporter reporter = new ConsoleErrorReporter(options);
       options.DafnyPrelude = "../../../../../Binaries/DafnyPrelude.bpl";
 
       var programString = @"trait Trait<A, B> { }";
-      ModuleDecl module = new LiteralModuleDecl(new DefaultModuleDefinition(), null);
-      Microsoft.Dafny.Type.ResetScopes();
-      BuiltIns builtIns = new BuiltIns(options);
-      Parser.Parse(programString, "virtual", "virtual", module, builtIns, reporter);
-      var dafnyProgram = new Program("programName", module, builtIns, reporter);
-      Main.Resolve(dafnyProgram, reporter);
+      var dafnyProgram = Utils.Parse(options, programString, false);
```

Comment:
Why is the parsing in Utils, but the Resolve is in Main? Could you move Parsing also in Main, or is there something I need to know?

Reply:
The Parse function is only used in tests, so it's in a testing `Utils` that's not in DafnyCore.

I'm hoping in the future there's be more consolidation, and everything can use `Compilation` to do their thing.